### PR TITLE
feat: Various improvements for pages linked from CFMM.

### LINKED ISSUE #458: Add screenshot of multi-tenant overview to 'Administration -> meshTenants'
The page
would greatly benefit from a screenshot showing a list of multi-cloud tenants to display the core value of it. It would also help users to more easily find the screen if they were to use it as reference documentation

### PR DESCRIPTION
**Pull request by Jelledb**
* Closes #458, #459, #461.

### CODE REVIEW

File: docs/administration.unmanaged-tenants.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+## Viewing unmanaged tenants
+
+Viewing unmanaged tenants can be easily done within the administration area. In the sidebar on the left, navigate to 'Platforms' -> 'Unmanaged Tenants'.
+
+A table will open up with all unmanaged tenants that are known to meshStack. This list is refreshed on a daily basis.
+
+It is also recorded when the unmanaged tenant was last observed. If the last observed date is from longer than a few days ago, it has dissappeared. This
+probably means that it was deleted and it no longer exists in the cloud platform.
+
+Tenants that are imported into the meshStack organizational model (meaning into a meshProject) are automatically removed from the list.
```
Comment (felixzieger) [inline docs/administration.unmanaged-tenants.md:26]: We say "to adopt a cloud tenant" when speaking about this.
```suggestion
Tenants that are adopted into meshStack are removed from umanaged tenants list and shown in the tenants list instead.
```

Comment (felixzieger) [reply docs/administration.unmanaged-tenants.md:26]: A link to "How to adopt an unmanaged tenant into meshStack" would be great (but I think there is no how-to content for this at the moment).

File: docs/meshstack.metadata-tags.md [hunk 2]
```diff
@@ -145,7 +145,10 @@ As a full example, a tag named `myCustomerTag` would be provided as an HTTP head
 
 ## Tags in Cloud Tenants
 
-Certain cloud platforms allow you to tag resources in order to help organize them. It is possible to use the metadata tags from meshStack to control how these resources are tagged during project replication. The whole lifecycle of these tags is managed. This means tags on the cloud platform are updated and removed depending on the underlying metadata of the replicated meshProject. In order to properly track meshStack managed tags, every tag is usually prefixed with a tag prefix (e.g. the `meshstack_` prefix in the tag label `meshstack_costcenter:12345`).
+Beside having tags in meshStack, it is also useful for cloud-native users to be aware of the metadata within e.g. Azure. This why meshStack
+supports the ability to "replicate" the tags into the actual cloud platforms. The entire lifecycle of these tags can be managed by meshStack. This means tags on the cloud platform are updated and removed depending on the underlying metadata of the replicated meshProject. In order to properly track meshStack managed tags, every tag is usually prefixed with a tag prefix (e.g. the `meshstack_` prefix in the tag label `meshstack_costcenter:12345`).
```
Comment (felixzieger) [inline docs/meshstack.metadata-tags.md:149]:
```suggestion
supports the ability to "replicate" the tags into the actual cloud platforms. The entire lifecycle of these tags can be managed by meshStack. This means tags on the cloud platform are updated and removed depending on the underlying metadata of the replicated meshProject. meshStack manages all tags of a prefixed namespace (e.g. the `meshstack_` prefix in the tag label `meshstack_costcenter:12345` indicates that this is a meshStack managed tag).
```

**Review by felixzieger — APPROVED** [review]

File: docs/administration.unmanaged-tenants.md (hunk 1 above)
Comment (Jelledb) [reply docs/administration.unmanaged-tenants.md:26]: I briefly mentioned the API for now. A how-to would be great in future ;-)

**Pull request merged**